website/static/js/filters.js

```diff
@@ -0,0 +1,124 @@
+(function($){
+    $.fn.filters = function (options) {
+        var settings = $.extend({
+            items: '.items'
+        }, options);
+
+        var itemsSelector = settings.items.join();
+
+        var filterConstraints = {};
+        var searchConstraints = {};
```

Comment:
So you expect there to be multiple search contraints, like multiple search boxes?